packages/ProgressIndicators/src/index.tsx

```diff
@@ -1 +1,2 @@
-export * from './ProgressBar';
+import ProgressBar from './ProgressBar';
+export { ProgressBar };
```

Comment:
If we'll have it this way then let us also export `ProgressBarProps`

Reply:
Agreed, let's also export the `ProgressBarProps` and the `defaultProgressBarProps` please 🙏

Reply:
Initially, I had added the export here: https://github.com/onaio/js-tools/blob/7acb12593d3554b322517fb5ca23604d1b3cd5e6/packages/ProgressIndicators/src/ProgressBar/index.tsx#L4-L19

Am not sure since this is new to me, is this any different?

Reply:
@RayceeM - As I see it, the issue here is around what exactly is being imported/exported in this file from the `ProgressBar.tsx` file, and how that effects how the code is referenced from outside the module.  

Currently this line:
```ts
import ProgressBar from './ProgressBar';
```
references this line:
```ts
export default ProgressBar;
```
which references this declaration:
```ts
class ProgressBar extends Component<ProgressBarProps, {}> { 🎉}
```
which I think technically _does_ expose `defaultProgressBarProps` through this declaration:
```ts
public static defaultProps = defaultProgressBarProps;
```
which references the original declaration of `defaultProgressBarProps`:
```ts
export const defaultProgressBarProps: Partial<ProgressBarProps> = { 🎊}
```
which is conveniently already being exported from the file. But what about the `ProgressBarProps` interface?

While it's currently being declared/exported from `/ProgressIndicators/src/ProgressBar/index.tsx`:
```ts
export interface ProgressBarProps { ✨}
```
it's not being imported/exported from `/src/ProgressBar/index.tsx`:
```ts
export { ProgressBar };
```
thus making the interface unaccessible when installed as a package.
```ts
import { defaultProgressBarProps } from '@onaio/progress-indicators'; 🚫 
import { ProgressBarProps } from '@onaio/progress-indicators'; 🚫 
import { ProgressBar } from '@onaio/progress-indicators'; ✅

const defaultProgressBarProps =  ProgressBar.defaultProps; ✅ 
type ProgressBarProps = ProgressBar.ProgressBarProps ⛔️ 
type ProgressBarProps = typeof defaultProgressBarProps; 🙅‍♂ (this only implies the typings based on the default prop types)
```

In my opinion all public/exported declarations should be accessible from either the package root file:
```ts
import { ProgressBar, defaultProgressBarProps,  ProgressBarProps } from '@onaio/progress-indicators';
```
OR from respective submodules:
```ts
import { ProgressBar } from '@onaio/progress-indicators'; 🎉 
const { defaultProgressBarProps,  ProgressBarProps } = ProgressBar; 🎊 ✨ 
```

Reply:
Got it! perfect explanation! thank you